### PR TITLE
feat(workflow): pre_save signal on User/Team for Actor

### PR DESCRIPTION
**Pull request by iProgramStuff**
When new users and teams are created, they need to have an actor. This pre_save signal should do the trick.

### CODE REVIEW

**Review by wedamija — COMMENTED** [review]
Looks good, let's just have a test to make sure this works as expected for users/teams

File: tests/sentry/models/test_actor.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from sentry.models import Actor, User
+from sentry.testutils import TestCase
+
+
+class ActorTest(TestCase):
+    def test_pre_save(self):
+        user = self.create_user(email="test@pre_save.com")
+        assert user.actor is not None
```
Comment (wedamija) [inline tests/sentry/models/test_actor.py:8]: Could you verify that actor is the correct type?

Comment (iProgramStuff) [reply tests/sentry/models/test_actor.py:8]: Yeah. But wouldn't the database insertion fail if it wasn't a FK to actor?

**Review by wedamija — APPROVED** [review]

File: tests/sentry/models/test_actor.py (hunk 1 above)
Comment (wedamija) [reply tests/sentry/models/test_actor.py:8]: I actually meant `Actor.type` is correct, words are hard

**Pull request merged**